package.json

```diff
@@ -104,11 +104,11 @@
     "jest-watch-typeahead": "1.0.0",
     "konn": "^0.7.0",
     "markdown-toc": "^1.2.0",
-    "nexus": "^1.1.0",
```

Comment:
Most of these make sense to me (updating Prisma version, etc) but why are we locking the version of Nexus? The latest version is 1.3.0

Reply:
The last version using GraphQL 16
When I ran the test i got type error

Reply:
Hm, I am surprised the dev dependencies aren't already managed by renovate. That should be done. In that case, all deps would be pinned exact.

Reply:
![image](https://user-images.githubusercontent.com/4535142/164532728-6cfa6391-5d93-440e-b0a8-0c03ff50618f.png)

Why did I update the packages from the first? 
https://github.com/prisma/nexus-prisma/runs/6078469084?check_suite_focus=true#step:4:7
<img width="903" alt="image" src="https://user-images.githubusercontent.com/4535142/164533257-a72236bd-0426-408b-928a-eb9f65c16914.png">

We got an error in action when installing packages @andrewicarlson said maybe we need to update the deps I did that and pushed then I get many issues I worked on them one by one as you see.